packages/openapi-react-query/package.json

```diff
@@ -65,17 +65,18 @@
     "openapi-typescript-helpers": "workspace:^"
   },
   "devDependencies": {
+    "@tanstack/react-query": "^5.59.16",
```

Comment:
Also we weren’t loading a specific version of `@tanstack/react-query` for tests, etc but we should be. Added as a `devDep` (users still control this package, but we still have to pin a version for our own local tests, etc.)